src/main/java/com/keyboardbuilder/enterprise/service/KeyBoardBuildStub.java

```diff
@@ -0,0 +1,18 @@
+package com.keyboardbuilder.enterprise.service;
+
+import com.keyboardbuilder.enterprise.dto.KeyboardBuild;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class KeyBoardBuildStub implements IKeyboardBuildService{
+
+    List<KeyboardBuild> allKeyboardBuild = new ArrayList<>();
+
+    @Override
+    public List<KeyboardBuild> fetchAll() {
+        return allKeyboardBuild;
+    }
+}
```

Comment:
Created KeyBoardBuildStub which implements IKeyboardBuildService interface. Also, added atrraylist and override to fetchAll() method which was declared in Service interface. It returns a value. The project did not had any Service class.

Source: Lecture Videos